web/pdf_viewer.js

```diff
@@ -730,7 +730,9 @@ class PDFViewer {
           } else if (isValidAnnotationEditorMode(mode)) {
             this.#annotationEditorUIManager = new AnnotationEditorUIManager(
               this.container,
-              this.eventBus
+              this.eventBus,
+              this.pdfDocument?.annotationStorage,
+              this.currentScale
```

Comment:
I suppose that there isn't a "simpler" way of obtaining the initial scale, inside of the editor code itself (e.g. using the `viewport`), without having to pass it in manually like this?